src/TobiasOlry/TalklyBundle/DependencyInjection/TobiasOlryTalklyExtension.php

```diff
@@ -15,6 +15,8 @@ public function load(array $config, ContainerBuilder $container)
             $container,
             new FileLocator(__DIR__ . '/../Resources/config')
         );
+
+        $loader->load('security.xml');
```

Comment:
Yeah, but it's a sub part of the service configuration and not all files in the config folder are service definitions. Alternatively you could put that file in a separate service folder, within the config folder